src/components/GroßerButton.js

```diff
@@ -0,0 +1,9 @@
+import "./GroßerButton.css";
+
+export default function GroßerButton({ text }) {
```

Comment:
It's okay if your content is in German, but your code should be in English. Simply because the programming language uses English words :)